CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 All notable changes to this project will be documented in this file.
 
 ---
+## [x.x.x]
+* Improvement:
+	* Set allowsExternalPlayback to false to avoid black screen when using AirPlay on Apple TV
```

Comment:
"Disable external playback to avoid black screen when using AirPlay on Apple TV" ?
no need to mention any parameter detail to changelog.